ReCaptcha/Assets/recaptcha.html

```diff
@@ -59,6 +59,14 @@
         post({ token });
         clearObservers();
       },
+      // 'expired-callback': () => {
+      //   post({ error: 28 });
+      //   clearObservers();
+      // },
```

Comment:
Why is this commented out? It looks like the code will never catch "expired response" errors

Reply:
Thanks for your review! I did debug with commented out this, and I forgot to fix it. I didn't mean to comment out, so I will fix it.